### PR TITLE
36034 on site notifications

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Description
On site notifications feature. This is hidden behind a feature flag and when activated will check VA Notify for any relevant notifications. Currently there is only debt notifications, but we expect this to be other types in the future. These are dismissible. 

## Testing done
Cypress tests done

## Screenshots
Screenshot 2022-05-17 103643
Screenshot 2022-05-17 103604

### CODE REVIEW

**Comment by cohnjesse** [comment]
@tmitchellgcio Would you be able to add a description of what this code does and some context around what this change is for? "on site notifications" doesn't tell me anything. about what this is, why its here, how I can see them, etc...

**Review by cohnjesse — COMMENTED** [review]
@tmitchellgcio Would you be able to add a description of what this code does and some context around what this change is for? "on site notifications" doesn't tell me anything. about what this is, why its here, how I can see them, etc...

**Comment by tmitchellgcio** [comment]
@cohnjesse no problem! Sorry about that. I just created it to see if the tests all pass, which they do locally, but for some reason not on here. I should have made it a draft until I get the tests passing.

**Comment by tlei123** [comment]
> to see if the tests all pass, which they do locally

@tmitchellgcio , to expose flakes, 20x-loop the root `describe(...)`:

```js
for (let i = 0; i < 20; i += 1) {
  describe(...); 
}
```

**Comment by tlei123** [comment]
@tmitchellgcio , since that CI-failing **Dashboard-Notifications E2E-spec** passes fine locally, i've added a CI-only skip to it so as to let this PR through.  Pls. keep an eye on the Checks list for the E2E test-results.

**Comment by tmitchellgcio** [comment]
> @tmitchellgcio , since that CI-failing **Dashboard-Notifications E2E-spec** passes fine locally, i've added a CI-only skip to it so as to let this PR through. Pls. keep an eye on the Checks list for the E2E test-results.

Thank you! @adamwhitlock1 @cohnjesse sorry for tagging you all earlier on this but it is now ready for review

**Comment by adamwhitlock1** [comment]
Was a ticket or support request put in to the testing tools team to investigate this? Might be something they could help with or identify what is going on in the CI for these cypress tests.

Other thought would be that maybe there is a cypress version mismatch between what the CI is using and what yall are running locally?

File: src/applications/personalization/dashboard/components/notifications/DebtNotification.jsx [hunk 1]
```diff
@@ -1,47 +1,34 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import moment from 'moment';
+import { VaAlert } from 'web-components/react-bindings';
```
Comment (adamwhitlock1) [inline src/applications/personalization/dashboard/components/notifications/DebtNotification.jsx:5]: I did just notice that there is a PR open to remove this alias (not sure exactly why its being removed), but might be good to just import directly from node modules via `@department-of-veterans-affairs/component-library/dist/react-bindings` so you don't have to go back and fix this later when that PR gets merged.

PR for reference: https://github.com/department-of-veterans-affairs/vets-website/pull/21127

**Review by adamwhitlock1 — APPROVED** [review]
LGTM

**Comment by tlei123** [comment]
> Was a ticket or support request put in to the testing tools team to investigate this?

@adamwhitlock1 , @tmitchellgcio [opened a tic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41688) for me to give it another try.  Will send on to Testing Tools Team if I still can't fix.

Thx for the version-mismatch idea!  I'll def. double-check that.  :)

**Pull request merged**